day-6/ques-2.js

```diff
@@ -0,0 +1,31 @@
+//Problem Statement: Create a class Shape with properties width and height and methods getArea(). Create two classes Rectangle and Triangle that inherit from the Shape class and implement the getArea() method for their respective shapes.
+
```

Comment:
solution is fulfill problem statement. getArea method should be part of shape class.

Reply:
resolved